src/expressions/plyql.pegjs

```diff
@@ -527,6 +537,8 @@ OrToken            = "OR"i             !IdentifierPart
 DistinctToken      = "DISTINCT"i       !IdentifierPart
 StarToken          = "*"               !IdentifierPart { return '*'; }
 
+AbsToken           = "ABS"i            !IdentifierPart { return 'abs'; }
+AbsoluteToken      = "ABSOLUTE"i       !IdentifierPart { return 'abs'; }
```

Comment:
by the logic outlined here AbsToken and AbsoluteToken should both return `'absolute'`